packages/tabs/src/TabsOverflow.ts

```diff
@@ -27,6 +27,7 @@ import {
 import { classMap } from '@spectrum-web-components/base/src/directives.js';
 import { ResizeController } from '@lit-labs/observers/resize-controller.js';
 import { Tabs } from './Tabs.js';
+import { Tab } from './Tab.js';
```

Comment:
```suggestion
import type { Tabs } from './Tabs.js';
import type { Tab } from './Tab.js';
```
I _think_ you can do this for `Tabs`, too, but if not, I retract my statement. Having these at `type` imports isolate the code inclusion to the TS context and _in some cases_ can reduce the final about of JS passed to the browser.